python/cuml/test/test_incremental_pca.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+import cupy as cp
+
+from sklearn.decomposition import IncrementalPCA as skIPCA
+
+from cuml.datasets import make_blobs
+from cuml.decomposition import IncrementalPCA as cuIPCA
+
+from cuml.test.utils import array_equal
+
+
+@pytest.mark.parametrize('nrows', [500, 5000])
+@pytest.mark.parametrize('ncols', [15, 25])
+@pytest.mark.parametrize('n_components', [10, 12])
+@pytest.mark.parametrize('sparse_input', [True, False])
+@pytest.mark.parametrize('density', [0.07, 0.4])
+@pytest.mark.parametrize('sparse_format', ['csr', 'csc'])
+def test_inverse_transform(nrows, ncols, n_components, sparse_input, density,
+                           sparse_format):
+
+    if sparse_format == 'csc':
+        pytest.skip("cupyx.scipy.sparse.csc.csc_matrix does not support"
+                    " indexing as of cupy 7.6.0")
+
+    if sparse_input:
+        X = cp.sparse.random(nrows, ncols, density=density,
+                             random_state=10, format=sparse_format)
+    else:
+        X, _ = make_blobs(n_samples=nrows, n_features=ncols, random_state=10)
+
+    cu_ipca = cuIPCA(n_components=n_components)
+    cu_ipca.fit(X)
+    cu_t = cu_ipca.transform(X)
+    cu_inv = cu_ipca.inverse_transform(cu_t)
+
+    sk_ipca = skIPCA(n_components=n_components)
+    if sparse_input:
+        X = X.get()
+    else:
+        X = cp.asnumpy(X)
+    sk_ipca.fit(X)
```

Comment:
any test of `partial_fit`?

Reply:
or batch size?

Reply:
I believe Divye and I briefly discussed testing different batch sizes offline. That should be a great test for partial_fit as well.